README.md

```diff
@@ -58,9 +58,35 @@ const useStoreWithUndo = create<StoreState>()(
 );
 ```
 
-## Convert to React Store
+## Then access `temporal` functions and properties of your store
 
-If you're using React, you can convert the store to a React hook using create from `zustand`.
+Your zustand store will now have an attached `temporal` object that
+provides access to useful time-travel utilities, including `undo`, `redo`, and `clear`!
+
+```tsx
+const App = () => {
+  const { bears, increasePopulation, removeAllBears } = useStoreWithUndo();
+  // See API section for temporal.getState() for all functions and
+  // properties provided by `temporal`
+  const { undo, redo, clear } = useStoreWithUndo.temporal.getState();
```

Comment:
This may be personal preference, but I think it is simpler in the docs to introduce (and recommend) `useStore.temporal.getState()` prior to something like `useTemporalStore`. I _think_ most users will not need to access member properties of the `temporal` object in a reactive way (that is, they'll likely just need `undo`, `redo`, `clear` and not need reactive access to `pastStates` or `currentStates`, `isTracking`, etc. I think introducing it this way may help facilitate a more straightforward mental model of what `zundo` is doing. But I'm not sure!

Reply:
I agree with this, thank you for making this change and for providing an explanation!